schemas/iso19139/src/main/plugin/iso19139/formatter/xsl-view/view.xsl

```diff
@@ -952,4 +988,55 @@
   <xsl:template mode="render-value"
                 match="@*"/>
 
-</xsl:stylesheet>
+  <!-- isBasedOn parentidentifier -->
+
+  <xsl:template mode="render-value"
+                match="gmd:parentidentifier/gco:CharacterString">
+    <span itemprop="isBasedOn">
+      <a href="{concat($nodeUrl, 'api/records/', .)}"><xsl:value-of select="."/></a>
+    </span>
+  </xsl:template>
+
+  <!-- license -->
+  <xsl:template mode="render-value"
+                match="gmd:MD_LegalConstraints/gmd:otherConstraints/gmx:Anchor/@xlink.href">
+    <span itemprop="license"><a href="{.}"><xsl:value-of select="."/></a></span>
+  </xsl:template>
+
+  <!-- spatialCoverage	Place 
+  temporalCoverage	DateTime -->
+  <xsl:template mode="render-value"
+                match="gmd:extent/gml:TimePeriod/gml:beginPosition">
+    <span itemprop="temporalCoverage" >
+      <xsl:value-of select="."/> / 
+      <xsl:value-of select="../endPosition"/>
+    </span>
+  </xsl:template>
+
+  <!-- thumbnailUrl	URL  -->
+  <xsl:template mode="render-value"
+                match="gmd:graphicOverview/gmd:MD_BrowseGraphic/gmd:fileName/gco:CharacterString">
+    <a href="{.}" itemprop="thumbnailUrl">
```

Comment:
Add the itemprop in https://github.com/geonetwork/core-geonetwork/blob/master/schemas/iso19139/src/main/plugin/iso19139/formatter/xsl-view/view.xsl#L117 so we have the overview with description in one place ? (and also rely on css rules for styling)